GPU/GPUCommon.cpp

```diff
@@ -1677,15 +1722,22 @@ void GPUCommon::Execute_TgenMtxNum(u32 op, u32 diff) {
 	const int end = 12 - (op & 0xF);
 	int i = 0;
 
-	while ((src[i] >> 24) == GE_CMD_TGENMATRIXDATA) {
-		const u32 newVal = src[i] << 8;
-		if (dst[i] != newVal) {
-			Flush();
-			dst[i] = newVal;
-			gstate_c.Dirty(DIRTY_TEXMATRIX);
-		}
-		if (++i >= end) {
-			break;
+	bool skipData = debugRecording_;
```

Comment:
Well, skipping the early/fast load of data.  I'll just invert it and call it `fastLoad`.

While *recording* (recording is not replay), we can't do any fast loads, because then it doesn't get recorded.  We need the data commands to be sent to the recorder.

-[Unknown]